### PR TITLE
Updated updateEORevents

### PR DESCRIPTION
**Pull request by glensee**

### CODE REVIEW

File: backend/lambda/updateEventOrRecurring/lambda_function.py [hunk 1]
```diff
@@ -172,6 +172,22 @@ def lambda_handler(event, context):
         recurring['pwas'] = {"data": recurring.pop("pwas_to_add", [])}
         recurring['volunteers'] = {"data": recurring.pop("vols_to_add", [])}
         data = {"recurring": recurring}
+    else:
```
Comment (arixpsy) [inline backend/lambda/updateEventOrRecurring/lambda_function.py:176]: Can add comment so i know this else handles which cases

File: frontend/graphql/event/UpdateEventOrRecurring.graphql [hunk 2]
```diff
@@ -14,13 +14,17 @@ mutation UpdateEventOrRecurring ($updateEventData: UpdateEventOrRecurringInput!)
 # Updating Single Event: 'frequency' in recurringData = 'None'
 # Repopulates event data based on fields in event
 #
-# Updating All Events: recurringData.'is_all' is True
-# Updates the specific values given within the recurringData
-# Adds/removes the vols and pwas based on the 4 fields within recurringData
+# Update This & Future Events: recurringData.'is_all' == 0
```
Comment (arixpsy) [inline frontend/graphql/event/UpdateEventOrRecurring.graphql]: I believe this is for updating single event that is recurring?

**Review by arixpsy — CHANGES_REQUESTED** [review]
see me comments

File: frontend/graphql/event/UpdateEventOrRecurring.graphql (hunk 2 above)
Comment (glensee) [reply frontend/graphql/event/UpdateEventOrRecurring.graphql]: I've updated, good eye. I sot

File: backend/lambda/updateEventOrRecurring/lambda_function.py (hunk 1 above)
Comment (glensee) [reply backend/lambda/updateEventOrRecurring/lambda_function.py:176]: Added!

**Review by arixpsy — APPROVED** [review]
look ok

**Pull request merged**